### DISCUSSION TITLE
How read and purge are supposed to operate

### DISCUSSION THREAD

**Discussion by adm2k** [FreshRSS/FreshRSS#5364] [Q&A] [answered]
I have a new install running (_and am new to Fresh RSS_) and I guess I don't understand how read and purge are supposed to operate. Basically I would like articles that have been read to "disappear" or be "purged" from view.  I am testing using one feed (https://chromeunboxed.com/feed/) and using all the standard settings. 

It seems sometimes a read article "disappears" from view or is purged but when "Update Feeds" is run all articles are showing again including articles from several days prior.

Again, I apologize if I am not understanding how this is supposed to work. All I want is when an article is read that it is removed from view and not show up again. Hope that makes sense.

Any help is much appreciated!

**Comment by math-GH** [comment, marked as answer]
Hi, you are welcome to use FreshRSS.

Have you seen this 2 buttons?
![grafik](https://user-images.githubusercontent.com/1645099/235324981-37ccecf6-ae1d-4166-90f8-102daf127d53.png)

With this buttons you can toggle your view: show/hide read articles (left button) and show/hide unread articles (right button)

Does it help you to understand?

**Comment by adm2k** [comment]
Yes this helps and thank you for your response!

**Comment by math-GH** [comment]
Nice. Have fun with FreshRSS :)
